### PR TITLE
Update Tyron

### PR DESCRIPTION
**Pull request by amygorgon**
I updated "ssityron" folder to "tyron" and also "project.md" and "logo" files. Thank you

### CODE REVIEW

**Review by Elliott-Green — APPROVED** [review]

**Comment by tralkan** [comment]
Hi @Elliott-Green, thanks for the approval, but who else are we waiting for before the merge? I don't see any requests.
We have an AMA with Zilliqa tomorrow, so we need this info updated. Thank you

**Comment by rrw-zilliqa** [comment]
Sorry this has taken so long!

**Pull request merged**
